runelite-client/src/main/java/net/runelite/client/plugins/driftnet/DriftNetPlugin.java

```diff
@@ -196,6 +214,22 @@ public void onInteractingChanged(InteractingChanged event)
 		}
 	}
 
+	private boolean checkFishOnTiles(NPC fish, Set<WorldPoint> adjacentTiles)
+	{
+		final WorldPoint fishTile = WorldPoint.fromLocalInstance(client, fish.getLocalLocation());
+		return adjacentTiles.contains(fishTile);
+	}
+
+	private boolean checkFishNextToNet(NPC fish, Iterable<DriftNet> nets)
+	{
+		return Iterables.size(Iterables.filter(nets, net -> checkFishOnTiles(fish, net.getAdjacentTiles()))) > 0;
+	}
```

Comment:
Instead of accepting an `Iterable`, a Collection or even a List would be preferable. That way, the Java stream API can be used instead of guava's `Iterables`:

```suggestion
	private boolean checkFishNextToNet(NPC fish, Collection<DriftNet> nets)
	{
		return nets.stream().anyMatch(net ->
		{
			final WorldPoint fishTile = WorldPoint.fromLocalInstance(client, fish.getLocalLocation());
			return net.getAdjacentTiles().contains(fishTile);
		});
	}
```